#! /bin/sh /usr/share/dpatch/dpatch-run
## r219 | relnev | 2003-07-11 03:54:45 +0200 (Fr, 11 Jul 2003) | 2 lines
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Another attempt at restoring the original EKG mode.

@DPATCH@
diff -urNad rott-1.0+dfsg~/rt_actor.c rott-1.0+dfsg/rt_actor.c
--- rott-1.0+dfsg~/rt_actor.c	2008-01-10 09:22:11.000000000 +0100
+++ rott-1.0+dfsg/rt_actor.c	2008-01-10 09:22:14.000000000 +0100
@@ -3362,9 +3362,13 @@
       else
          {
          MISCVARS->randgibspeed = true;
-//         SpawnParticles(ob,GUTS,75);
-//MED
+
+#ifdef MEDGIBS
          SpawnParticles(ob,GUTS,150);
+#else
+         SpawnParticles(ob,GUTS,75);
+#endif
+         
          MISCVARS->randgibspeed = false;
          }
       SpawnParticles(ob,GUTS,40);
@@ -3847,10 +3851,12 @@
                      (randadj<<4);
          dz = 100 + (randadj<<3);
 
-//MED
-//         nspeed = 0x2800 + (randadj<<7);
+#ifdef MEDGIBS
          nspeed = 0x2800;
-         
+#else
+         nspeed = 0x2800 + (randadj<<7);
+#endif
+
          randphi = atan2_appx(FindDistance(dx,dy),dz<<10);
          }
 
@@ -3872,8 +3878,11 @@
       new->temp3 = (MISCVARS->gibgravity == -1)?(GRAVITY):(MISCVARS->gibgravity);
 
       new->speed = nspeed>>1;
-      //if (MISCVARS->randgibspeed == true)
-      //   new->speed += (randadj << 11);
+
+#ifndef MEDGIBS
+      if (MISCVARS->randgibspeed == true)
+         new->speed += (randadj << 11);
+#endif
 
 //      if (ob->state == &s_snakefireworks)
          new->z = ob->z;
